src/js/components/Banner/ShabadavaliBanner.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Banner from './Banner';
+
+const ShabadavaliBanner = () => {
+
+  const {
+    isFetchingData: isFetchingBannerMessage,
+    data: bannerData,
+  } = {
+    isFetchingData: false,
```

Comment:
@MySmilingTurban why are we creating this object here with `isFetchingData` property when we already knew it's false ?